test/units/modules/network/ovs/test_openvswitch_bridge.py

```diff
@@ -155,6 +173,16 @@ def test_openvswitch_bridge_present_creates_fake_bridge(self):
         self.execute_module(changed=True, commands=commands,
                             test_name='test_openvswitch_bridge_present_creates_fake_bridge')
 
+    @pytest.mark.usefixtures('patched_openvswitch_bridge')
+    def test_openvswitch_bridge_updates_vlan(self):
```

Comment:
Are you sure it doesn't work with just
```suggestion
    def test_openvswitch_bridge_updates_vlan(self, patched_openvswitch_bridge):
```
?

Reply:
Oh.... It derives from `unittest.TestCase` which is almost incompatible with pytest..